### PR TITLE
feat: Update the Code of Conduct to the latest version

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Update the Code of Conduct to the latest version.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Violating these terms may lead to a permanent ban.
 ### 4. Permanent Ban
 
 **Community Impact**: Demonstrating a pattern of violation of community
-standards, including sustained inappropriate behavior, harassment of an
+standards, including sustained inappropriate behavior,  harassment of an
```
Comment (schmelto) [inline CODE_OF_CONDUCT.md:109]:
```suggestion
standards, including sustained inappropriate behavior, harassment of an
```

**Review by schmelto — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Panquesito7) [reply CODE_OF_CONDUCT.md:109]: I just copied it from the Contributor Covenant code of conduct page, not sure if we should modify it. 🤷‍♂️

Comment (schmelto) [reply CODE_OF_CONDUCT.md:109]: I think a double space could be modified

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
@Panquesito7 could you check why the github action is failing here think its a prettier issue

**Pull request merged**